pages/api/teams/[slug]/invitations.ts

```diff
@@ -127,6 +132,8 @@ const handleDELETE = async (req: NextApiRequest, res: NextApiResponse) => {
 
   await sendEvent(teamMember.teamId, 'invitation.removed', invitation);
 
+  recordMetric('invitations.deleted');
+
```

Comment:
Can we keep the naming consistent. `invitation.removed` instead of `invidation.deleted`. Please check for more such things.